lib/lotus/model/adapters/sql/query.rb

```diff
@@ -663,6 +660,11 @@ def apply(query)
             end
           end
 
+          def _push_to_conditions(condition_type, condition)
+            raise ArgumentError, 'You need to specify a condition.' unless condition
```

Comment:
@guilhermeFranco Can we change with the following code?

```ruby
raise ArgumentError.new('You need to specify a condition.') if condition.nil?
```

The reasons are:
  1. Better style, passing a message to an instance of an exception is more OOP
  2. In my experience falsey check with Ruby can be source of errors. Explicit `nil` check is more explicit, less bug prone and faster.

Can you please also add API docs for it? Thank you! :+1: